app/test/shared/handlers_test_utils.dart

```diff
@@ -19,6 +19,11 @@ Future expectApiException(Future future,
         (e) => e.bodyAsJson(),
         'bodyAsJson',
         {
+          'error': {
+            'code': code ?? isNotNull,
+            'message': message == null ? isNotNull : contains(message),
+          },
+          // TODO: remove after the above gets deployed live
```

Comment:
Curious, why?

Reply:
Why keep it duplicated?

Reply:
Oh, curious why you keep it duplciated for now, instead of just removing it? (I'm probably missing some reason).

Reply:
There is a chance that somebody loaded the page, we switch traffic to new version, they submit the form, it returns an error, and our frontend is unable to process it because the JSON content is not where we are expecting... Which reminds me: we'll need to change the frontend to read the `{'error': {'message': 'XX'}}` too.

Reply:
PTAL on the last commit